README.md

```diff
@@ -145,7 +148,20 @@ As desired, repeat this process for any additional assets.
 
 ## Labeling a Video
 
-Coming soon!
+Labeling a video is much like labeling a series of images. When you select a video from the left, it will begin automatically playing, and you have several controls on the player, as seen here:
+![alt text](docs/images/video-player.png "Video Player")
+
+Besides the normal video playback controls, you will see two extra pairs of buttons. 
+
+On the left, you will see the previous and next frame buttons. Clicking these will pause the video, and move to the next appropriate frame as determined by your project settings. For example, if your project settings have a frame extraction rate of 1, these buttons will be moved back or forward 1 second, where if your frame extraction rate was 10, you will be moved back or forward a tenth of a second.
+
+On the right, you will see the previous and next tagged frame buttons. Clicking these will pause the video and move to the next or previous frame that has a tagged region on it, if it exists. If no tagged frame exists to navigate to, it will pause the video and do nothing else.
```

Comment:
"On the right are the previous and next tagged frame buttons."
"...move to the next or previous frame that has a previously tagged region on it, if a tagged frame exists."